### PR TITLE
DEP Remove unsupported modules

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/342

### CODE REVIEW

File: phpunit.xml.dist
Comment (ChloeSartorelli) [inline phpunit.xml.dist]: These recipes aren't around anymore. These should be restructured so that either:

a) similar modules are grouped together in test suites, or
b) test suites are grouped to include roughly the same number of modules, or
c) test suites are grouped so that unit tests take roughly the same amount of time per test suite.

e.g. having hybridsessions in its own suite is a waste of a runner.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Need to update `.cow.json`

File: phpunit.xml.dist
Comment (emteknetnz) [reply phpunit.xml.dist]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM, will merge once all other PRs have been merged

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Need to update list of behat tests in ci.yml (I was wondering why there was still red here lol)

**Comment by emteknetnz** [comment]
Updated

File: .github/workflows/ci.yml
Comment (ChloeSartorelli) [inline .github/workflows/ci.yml]: Keep the ones we're making CMS 6 compatible, or else how will we know they're compatible?

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml
Comment (emteknetnz) [reply .github/workflows/ci.yml]: Updated, and add subsites back in too

**Comment by GuySartorelli** [comment]
Userforms CI failure is a known issue and not related to this PR

**Review by GuySartorelli — APPROVED** [review]
LGTM, just letting the CI finish but no failures there should be related to this PR in any case.

**Pull request merged**
